### ISSUE TITLE
Mapnik iOS

### ISSUE THREAD

**Issue by ccarse** [mapnik/mapnik-support#14]
So, say I have a mapnik xml that specifies a sql datasource and some styles. And I have Mapnik running on iOS. Given an x and y tile index and a zoom level, how would I generate a 256x256 image using Mapnik?

**Comment by ccarse** [comment]
Okay, so I got it working. Performance isn't great but it works! Can't wait for the code you guys are about to release. Right now, I'm using the new overlay apis in iOS 7, subclassing MKTileOverlay to return a tile rendered on the fly by Mapnik. So, sqlite->mapnik->image tiles->mapkit. Looking in Inspector, it looks like it's not Mapnik that's slow, but that wasn't looking too hard. 

![ios simulator screen shot jul 23 2013 4 15 30 pm](https://f.cloud.github.com/assets/1965943/844543/f264d332-f3d4-11e2-9ab8-427475137ad1.png)

**Comment by springmeyer** [comment]
glad you got it working. the code samples I have that I plan to publish are not iOS 7 however since that is not a public OS of course.

**Comment by ccarse** [comment]
Of course. Just a heads up, perf was horrible in the simulator but on the device it runs great. Thanks again!

**Comment by Whirlwind** [comment]
Hi, I am sorry. Can somebody tell me how to use it in iOS ?

**Comment by Whirlwind** [comment]
Hi @ccarse , Can you give me some code to how to use it in iOS? I need it. Thanks very much. :)

**Comment by obrhoff** [comment]
I'm just looking in into this topic. Are there some more resources to get me started?

**Comment by obrhoff** [comment]
Well It seems I got it working but I'm having trouble to convert MKTileOverlayPath to the right box2d.

**Comment by alperr** [comment]
I am also interested in this . 
Any resource or code samples about 256x256px tile generation from sqlite data would be appreciated
